lib/modules/datasource/types.ts

```diff
@@ -31,8 +30,7 @@ export interface GetPkgReleasesConfig {
   registryUrls?: string[] | null;
   additionalRegistryUrls?: string[];
   datasource: string;
-  depName: string;
-  packageName?: string;
+  packageName: string;
```

Comment:
The two changes in this file are essentially the "root" change and everything else flows from there.

Reply:
simplifies datasources